public/main.js

```diff
@@ -417,6 +493,8 @@ medVitd.addEventListener('click', (event) => {recordEvent('meds', 'vitamin d')})
 // requests to Firebase. Try using the local cache instead.
 entriesFilter.addEventListener('change', (event) => {clearEntries(); loadRecentData()});
 timeFilter.addEventListener('change', (event) => {clearEntries(); loadRecentData()});
+// A click anywhere outside the CSV textarea should hide it.
+document.onclick = function(){ csvTextarea.setAttribute('hidden', true); };
```

Comment:
Since it's no longer possible to show again after it's hidden, can you delete the element rather than hide it so it's not a huge thing hidden in the html? That'd be something like csvTextarea.removeAllChildren() or csvTextarea.textContent = '';